classes/genre.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'music_album'
+
+class Genre
+  attr_accessor :name
+
+  def initialize(name)
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(album)
+    @items << album
+    album.genre = self
```

Comment:
As shown in the diagram kindly change the genre method to add_genre

Reply:
Alright working on it.